src/js/organisms/filter-form.js

```diff
@@ -29,43 +29,76 @@ module.exports = class FilterForm {
     this.state = parseQueryParams();
 
     // Elements for the filter form/list.
+    // IMPORTANT: there are TWO forms in any page featuring this component: one for
+    // desktop viewports and one for mobile/collapsed-menu filtering
```

Comment:
ouch! Do we really need two? It might be a necessary evil, but I thought I would ask.

Reply:
Again, an inherited setup, which could probably be addressed be refactoring it all with flexbox and flex-direction, but it's not been a big priority (and I doubt we have scope right now, but maybe in a couple of months)